ios/Runner/Info.plist

```diff
@@ -2,6 +2,11 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>CFBundleLocalizations</key>
+	<array>
+		<string>en</string>
+		<string>es</string>
+	</array>
```

Comment:
Setting this up in Xcode is weird because they have a limited set of dropdown values. But you can type your own directly. Do we need to add es-419 for Latin American Spanish? Seems to work either way, so idk what this really adds. Just doing it cause they docs say so. https://docs.flutter.dev/development/accessibility-and-localization/internationalization#localizing-for-ios-updating-the-ios-app-bundle